releng/com.b2international.snowowl.server.update/.launch/snow-owl-oss.launch

```diff
@@ -75,7 +75,8 @@
         <setEntry value="org.antlr.runtime@default:default"/>
         <setEntry value="org.apache.commons.codec@default:default"/>
         <setEntry value="org.apache.commons.io@default:default"/>
-        <setEntry value="org.apache.commons.lang3@default:default"/>
+        <setEntry value="org.apache.commons.lang3*3.1.0.v201403281430@default:default"/>
+        <setEntry value="org.apache.commons.lang3*3.7.0@default:default"/>
```

Comment:
Two versions of `org.apache.commons.lang3` are now selected in this launch configuration.